Project.toml

```diff
@@ -20,9 +20,9 @@ XMLDict = "228000da-037f-5747-90a9-8195ccbf91a5"
 
 [compat]
 AWSCore = "0.5, 0.6"
-AWSS3 = "0.6"
+AWSS3 = "0.6, 0.7"
```

Comment:
We should not include `0.7`, because that is using `AWS.jl` in the backend.
We would need to add in `AWS.jl` as well here, but that leads to headaches in the namespace for `AWSConfig`.

I think keeping this on `0.6` and using `AWSCore` for now is fine. And in the future migrate this to `AWS.jl` and support only `AWSS3@0.7`.

But in reality this entire package is just a means to an end of getting `AWSBatch` and `AWSClusterManagers` up, and we can modify those packages to just use `AWS.jl` and archive this package.

Reply:
huh I don't even remember why I bumped that.. ran the tests locally and they pass without it.

Reply:
I *believe* you need to manually do:

```
] up AWSS3
```

And then the tests probably would have exploded.